src/main/java/com/maddyhome/idea/vim/group/IjVimStorageService.kt

```diff
@@ -15,22 +15,20 @@ import com.maddyhome.idea.vim.ex.ExException
 import com.maddyhome.idea.vim.newapi.ij
 
 internal class IjVimStorageService : VimStorageServiceBase() {
-  override fun <T> getDataFromEditor(editor: VimEditor, key: com.maddyhome.idea.vim.api.Key<T>): T? {
+  override fun <T> getDataFromWindow(editor: VimEditor, key: com.maddyhome.idea.vim.api.Key<T>): T? {
```

Comment:
Hmm. This is two abstractions coming from different directions. I like that the storage API reflects Vim terminology, in that options and variables are stored "local to window", but `VimEditor` is an abstraction over IntelliJ's own editor. 

I'm not sure on the correct abstraction for Vim's "window". I don't remember seeing "editor" used much in the Vim docs, and Vim's screens are split into windows, which is a view on a buffer that can be edited. In other words, I don't think Vim has the concept of "editor". But it would be weird for a `VimWindow` to have editor functions. Perhaps we should have `VimWindow` as a base interface for `VimEditor`, so an editor "is a" window, but a window doesn't have surprising editor methods. I don't think it's worth changing in this PR, but something to think about for another grand refactoring.